src/pydantic_zarr/v3.py

```diff
@@ -311,13 +334,19 @@ def from_zarr(cls, array: zarr.Array) -> Self:
 
         """
         from zarr.core.metadata import ArrayV3Metadata
-        from zarr.core.metadata.v3 import V3JsonEncoder
 
         if not isinstance(array.metadata, ArrayV3Metadata):
             raise ValueError("Only zarr v3 arrays are supported")  # noqa: TRY004
-        meta_json = json.loads(
-            json.dumps(array.metadata.to_dict(), cls=V3JsonEncoder), object_hook=tuplify_json
-        )
+        try:
```

Comment:
I'd have a preference for an expclit version check here, instead of try/except, so the case where zarr-python 3.0.x is installed but the install is broken with an ImportError, the ImportError still surfaces instead of being caught.